shared/js/background/devtools.es6.js

```diff
@@ -24,7 +24,7 @@ function connected (port) {
             const matchedTracker = trackers.getTrackerData(requestData.url, siteUrl, requestData)
             if (tracker.matchedRule) {
                 // find the rule for this url
-                const ruleIndex = matchedTracker.tracker.rules.findIndex((r) => r.rule.toString() === tracker.matchedRule)
+                const ruleIndex = matchedTracker.tracker.rules.findIndex((r) => r?.rule?.toString() === tracker.matchedRule)
```

Comment:
rule itself can be null? This sounds like an error in data, should we hide it here?

Reply:
r shouldn't be no, will remove the first ?

Reply:
You haven't removed the first `?` :)